platform/security/sts/security-sts-ldaplogin/src/main/java/ddf/ldap/ldaplogin/SslLdapLoginModule.java

```diff
@@ -127,16 +133,26 @@
 
     private LDAPConnectionFactory ldapConnectionFactory;
 
+    private ServiceReference serviceReference;
+
+    public SslLdapLoginModule() {
```

Comment:
Didn't notice this before, but why did we add this? This is going to cause initialize to get called twice.